File: generator/output/ts-model/auth2.model.ts
class Auth2Model {
  public id: number = 0;

  // deleted 
  public deleted?: number = 0;

  // unionid unionid
  public unionid?: string;

  // user_id 
  public user_id: number = 0;

  // uuid uuid
  public uuid: string;

  // source 认证来源
  public source: string;

  // raw_data 
  public raw_data: string;

  public created_at: string;

  public updated_at: string;
}

const Auth2ModelFields = {
  id: "id",
  deleted: "deleted",
  unionid: "unionid",
  user_id: "user_id",
  uuid: "uuid",
  source: "source",
  raw_data: "raw_data",
  created_at: "created_at",
  updated_at: "updated_at"
};

export {Auth2Model, Auth2ModelFields};

File: backend/src/main/java/com/licenseair/backend/domainModel/Auth2Model.java
package com.licenseair.backend.domainModel;

import java.util.List;
import lombok.Data;

import java.lang.String;
import java.lang.Long;
import java.lang.Integer;
import java.sql.Timestamp;

/**
* Created by licenseair.com
* Auth2
*/
@Data()
public class Auth2Model {
  public Long id = null;

  /**
  *
  */
  public boolean deleted = false;

  /**
  * unionid
  */
  public String unionid = null;

  /**
  *
  */
  public Long user_id = null;

  /**
  * uuid
  */
  public String uuid = null;

  /**
  * 认证来源
  */
  public String source = null;

  /**
  *
  */
  public String raw_data = null;

  public Timestamp created_at = null;

  public Timestamp updated_at = null;

}
